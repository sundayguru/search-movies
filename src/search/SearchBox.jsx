import React from 'react'

const SearchBox = ({filterResult, search}) => (
    <div className="search-container">
        <input type="text" className="search-box" placeholder="Enter movie name..." onChange={(e) => {
            filterResult(e.target.value)
        }} value={search} />
    </div>
)


export default SearchBox