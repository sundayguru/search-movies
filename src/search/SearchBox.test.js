import React from 'react'
import SearchBox from './SearchBox'
import renderer from 'react-test-renderer'


test('<SearchBox />', () => {
    const component = renderer.create(
        <SearchBox search="spider" movies={[]} />,
    );
    
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
});
