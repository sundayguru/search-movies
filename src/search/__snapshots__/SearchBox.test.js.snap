// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchBox /> 1`] = `
<div
  className="search-container"
>
  <span>
    <input
      className="search-box"
      disabled={false}
      onBlur={[Function]}
      onChange={[Function]}
      onKeyDown={[Function]}
      placeholder="Enter movie title..."
      value=""
    />
  </span>
</div>
`;
